src/binary.ts

```diff
@@ -255,6 +255,35 @@ export class Binary {
     }
     return new Binary(data, type);
   }
+
+  /** @internal */
+  [Symbol.for('nodejs.util.inspect.custom')](): string {
```

Comment:
`Symbol.for('nodejs.util.inspect.custom')` won't work with the version of `inspect` from browserify (which we use in the browser shell). Would also have some issue with older node.js versions, which we might not need to support though.

I think something like:

```
let inspectMethodName = Symbol.for('nodejs.util.inspect.custom');

if (!inspectMethodName) {
   try {
      // note that we can't detect much here cause webpack / parcel etc may mess with it
      // so we just try.
      inspectMethodName = require('util').inspect.custom;
   } catch (e) {
      inspectMethodName = 'inspect';
      // ...
   }
}
```

Reply:
@mcasimir This does provide a plain `inspect()` method, but I think it’s good practice to also define the symbol-based property if one provides the named variant

Reply:
Oh i see what you mean: `require('util').inspect.custom` is always === 'inspect'?

Reply:
Uh … no, it’s always a symbol if it exists

What I mean is that in this PR, both the symbol variant and the plain `inspect() { ... }` variant are added, so it’s always going to work

Reply:
With browserify too? I mean i think with browserify it very well be the string 'inspect'. I guess we can just read the code there. I'm sure few months ago was not`Symbol.for('nodejs.util.inspect.custom')` but if we are lucky and is always inspect we are good.

Reply:
Ok: https://github.com/browserify/node-util/blob/0915f593c45eb131e3e7f4718e0b4861433acaff/test/node/inspect.js#L155

I now i understand, so browserify is using `inspect` directly, which is probably what was happening in some old node. Makes sense, so we don't need that weird hack.